### DISCUSSION TITLE
Batching algorithm for multiple orders

### DISCUSSION THREAD

**Discussion by krutarth4** [krutarth4/SHYFTX#52] [Ideas]
There might be question on how we are doing batching of orders from farmers. We need to think about what makes more sense from truckers POV.

**Comment by krutarth4** [comment]
@KindTechs @vahidjb @Nnvvy96 

My first idea , was that we let the truckers decide while registering a radius only in which they are supposed to get batch order or the dropoff points and pickup points needs to be under 50km from each other as an example.


Would be nice if we brainstorm some ideas for it

**Reply by vahidjb** [reply]
Umm..This is how I see it: For real development until presentation, keep it simple. However, I don't think your solution sounds good for presenting data to the referee. This course focuses on sustainable business development, so it would be wise to refer to this point multiple times in the presentation. Therefore, in the presentation, we should mention that the batching system uses data analysis to create a solution that can significantly reduce CO2 emissions and costs.

There are some elements we need to consider:
- Pickup Locations / Delivery Locations radius 
- Time Windows overlap
- Goods Compatibility
- Cost Thresholds
- Emission Reduction Targets
- Capacity 

I think this way sounds more professional.

**Reply by krutarth4** [reply]
This is only for the Q/A if asked : not for the implementation, just so that we at least have a little knowledge about it. 

@vahidjb maybe you can look into specifics for it, like car pooling would be one similar use case.

**Reply by vahidjb** [reply]
@krutarth4 Ok will do it.
Also I added some notes on Figma about CO2 calculation logic. Not for implementation just to know it. Plz check it as well
